torch/export/exported_program.py

```diff
@@ -994,16 +1000,33 @@ def run_decompositions(
         `Core ATen Operator Set <https://pytorch.org/docs/stable/torch.compiler_ir.html>`_.
 
         For now, we do not decompose joint graphs.
+
+        If you want to disable a decomposition for certain operator, you can do following:
+
+        .. code-block:: python
+
+            ep = torch.export.export(model, ...)
+            ep = ep.run_decompositions(decomp_table={your_op: None})
+
+        If you want to get a core aten operator set except for certain operator, you can do following:
+
+        .. code-block:: python
+            ep = torch.export.export(model, ...)
+
+            from torch._decomp import core_aten_decompositions
+            decomp_table = core_aten_decompositions()
+            decomp_table[your_op] = None
```

Comment:
What's the difference between this and removing it from `decomp_table` (like, literally `pop`)?

If no difference, maybe it will be simpler to say `None` only works for CIA.

(Also, I don't see `decomp_table` passed in here! Is this a side effect on a global table?)